Lib/asyncio/selector_events.py

```diff
@@ -1063,7 +1066,46 @@ def write(self, data):
         self._buffer.append(data)
         self._maybe_pause_protocol()
 
-    def _write_ready(self):
+    def _write_sendmsg(self):
+        assert self._buffer, 'Data should not be empty'
+        if self._conn_lost:
+            return
+        try:
+            n = self._sock.sendmsg(self._buffer)
```

Comment:
`sendmsg` cannot work with more than `SC_IOV_MAX` buffers.
On Linux the value is 1024 AFAIK, can be retrieved by `sysconf()`.